docs/setup-cluster/deploy-cluster/on-prem/docker.rst

```diff
@@ -142,24 +144,29 @@ To start an agent container with environment variables instead of a configuratio
        -e DET_MASTER_PORT=8080 \
        determinedai/determined-agent:VERSION
 
-When agents are on different machines than the master, the Determined master hostname or IP should
-just be how you expect your agent machines to reach the master machine.
-
-However, if the agents are on the same machine as the master, ``127.0.0.1`` will typically not work,
-unless both the master and agent containers were started with ``--network host``. Note that even
-that in that case, you will also have to configure workload containers to use ``host`` network mode,
-as described :ref:`below <network-host>`. Otherwise, if the master machine has a static IP address
-from your router, you can use that. What is important is that it is reliably addressable from both
-inside and outside of Docker containers (since the Fluent Bit container will always use host
-networking).
-
-Determined internally makes use of `Fluent Bit <https://fluentbit.io>`__. The agent uses the
-``fluent/fluent-bit:1.9.3`` Docker image at runtime. It will attempt to pull the image
-automatically; if the agent machines in the cluster are not able to connect to Docker Hub, the image
-must be manually placed on them before Determined can run. In order to specify a different image to
-use for running Fluent Bit (generally to make use of a custom Docker registry---the image should not
-normally need to be changed otherwise), use the agent's ``--fluent-logging-image`` command-line
-option or ``fluent_logging_image`` config file option.
+.. note::
+
+   **Agents and Master on Different Machines**: If your agents and master are on different machines,
+   the Determined master hostname or IP address should be set to a value that allows your agent
+   machines to connect to the master machine.
+
+   **Agents and Master on the Same Machine**: If your agents and master are on the same machine,
```

Comment:
Now that these are contained within the paragraphs, I think not overly capitalizing them looks better:
```suggestion
   **Agents and master on different machines**: If your agents and master are on different machines,
   the Determined master hostname or IP address should be set to a value that allows your agent
   machines to connect to the master machine.

   **Agents and master on the same machine**: If your agents and master are on the same machine,
```